requirements-dev.txt

```diff
@@ -21,3 +21,6 @@ einops # required for MPT
 openai
 requests
 ray
+
+# Benchmarking
+aiohttp==3.9.3
```

Comment:
Why is the version fixed? Can we just simply use `aiohttp` without a specific version?

Reply:
I think using `aiohttp` without a specific version should be okay since we don't use anything special other than the `ClientSession` and `ClientTimeout` which shouldn't have breaking changes very often.